### DISCUSSION TITLE
AYUDA  VUE JS no puedo correr " npm run serve "

### DISCUSSION THREAD

**Discussion by Jorge-Macias-Cruz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#556] [Q&A] [answered]
Hola saben porque no puedo hacer correr el comando npm run serve ?

Estoy en MacOS 

al abrir el proyecto me salen estos warnings 
me aparece algo referente a la extensión Vetur

![Captura de pantalla 2022-03-29 a la(s) 14 29 30](https://user-images.githubusercontent.com/37346944/160701620-92dd0f1e-e767-4956-b2c7-5bd7ca9bcc9a.png)

y a la hora de correr el comando me sale estos errores:
![Captura de pantalla 2022-03-29 a la(s) 14 33 47](https://user-images.githubusercontent.com/37346944/160702037-2d48c092-2a45-4fdc-9110-c5cb362d7c6f.png)

**Comment by hackit92** [comment]
Que tal, en algun momento moviste o renombraste la carpeta del proyecto? Veo que tiene un espacio, a veces eso puede interferir en la correcta ejecución de un comando. Esos 2 puntos son lo que por el momento se me ocurre

**Comment by doguedogue** [comment]
Ya corriste 
Npm install ?


Colócate en la ruta principal de tu proyecto dónde ves el archivo package.json

y ahí ejecuta 
Npm run serve

A veces en Mac también usan Yarn en vez de npm.

**Reply by SamaelGT** [reply]
Hola yo tambien me quede atorado en ese, y aun asi utilizando el npm install da un error, y el termino yarn no me da permitido, utilizo el sudo para realizar operaciones en la terminal.

**Reply by Jorge-Macias-Cruz** [reply]
ya lo intente asi como me mencionas con npm install y solo se crea un nuevo package-lock.json 

y tambien al opsicionarme en package.json y correr el comando y tampoco sigo con los mismos errores 

yarn no me reconoce

**Comment by AngelCruzO** [comment]
No conozco Mac, pero si no veo mal, creo que no está ingresando en la carpeta del proyecto

**Comment by SamaelGT** [comment]
si gustas pasar a la discussion que habia abierto, ahi me estan ayudando, un buen compañero @carlos0318 👍🏽 
https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/545

**Comment by alejandronjj** [comment]
Ahí dice que no encuentra algunos archivos. Lo más probable es que estés ejecutando el comando NPM en otra carpeta. Ingresa a tu carpeta vue-launchx y ejecuta el comando ahí

**Reply by Jorge-Macias-Cruz** [reply]
si lo realice ya pude correr el comando de npm run serve desde dentro de la carpeta vue-launchx
pero me sigue marcando problema en la parte de  App.vue  en el "helloworld"

**Comment by Jorge-Macias-Cruz** [comment, marked as answer]
lo que me funcionó fue

cd vue-launchx
npm install
npm run serve 

y ya pude obtener el servidor como el MC en el localhost:8080

**Reply by SamaelGT** [reply]
cambia el npm run serve por npm run dev, a mi me esta funcionando con ese, y me da un localhost:3000

**Reply by Jorge-Macias-Cruz** [reply]
si compañero también eso me funciono

**Reply by SamaelGT** [reply]
por nada a mi tambien me ayudo un compañero 👍🏽

**Comment by OmarVenturaP** [comment]
verifica eso que comentan los compañeros, es la carpeta, debes abrirla directamente a VSC, a mi me paso igual, solo así se compuso
